app/static/src/scss/includes/_browse-consignment.scss

```diff
@@ -0,0 +1,245 @@
+.consignment-filter {
+    &-container {
```

Comment:
are all of these styles unique to the consignment filter, or are some similar to the other filter pages? If so, we should reuse/ share styles that are used between multiple components

Reply:
I think most of them being added by Candace and Colin, need to be revisited during refactoring and make it one generic style which can be used across different browse views

Reply:
okay, if we could, it might be easier to refactor as we go but dont want to block this PR